ci/test_wheel.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+RAPIDS_PY_VERSION="312" RAPIDS_PY_WHEEL_NAME="dask-cuda" rapids-download-wheels-from-s3 ./dist
```

Comment:
```suggestion
RAPIDS_PY_WHEEL_NAME="dask-cuda" RAPIDS_PY_WHEEL_PURE="1" rapids-download-wheels-from-s3 python ./dist
```

Let's match the way this is spelled in other pure-Python cases like:

* `jupyterlab-nvdashboard`: https://github.com/rapidsai/jupyterlab-nvdashboard/blob/d9746aefaab945241461046fb9d87334339c13a2/ci/test_wheel.sh
* `cudf-polars`: https://github.com/rapidsai/cudf/blob/a95fbc88f94df24c3418766fbbea5b6633ff2328/ci/test_wheel_cudf_polars.sh#L9

Keeping these scripts looking similar is helpful when we do touch-all-the-repos types of migrations.